.claude/scripts/context-monitor.py

```
@@ -31,6 +31,51 @@ def get_queue_status() -> str:
+    Args:
+        transcript_path: transcript JSONL 파일 경로
+
+    Returns:
+        {"used": int, "limit": int} 또는 None
+    """
+    if not transcript_path or not Path(transcript_path).exists():
+        return None
+
+    try:
+        # 최근 20줄만 읽기 (성능 최적화)
+        with open(transcript_path, 'r', encoding='utf-8') as f:
```

Ruff (0.14.3)

Unnecessary mode argument

Remove mode argument

(UP015)